pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -114,3 +118,40 @@ func (e *Executor) checkApproval(ctx context.Context) (string, bool) {
 	}
 	return approveCmd.Commander, true
 }
+
+func (e *Executor) reportDeploymentApproved(ctx context.Context) {
+	ds, err := e.TargetDSP.Get(ctx, e.LogPersister)
+	if err != nil {
+		e.LogPersister.Errorf("Failed to prepare running deploy source data (%v)", err)
+	}
+
+	var (
+		approvers []string
+
+		relPath  = e.Application.GitPath.GetDeploymentConfigFilePath()
+		repoPath = ds.RepoDir
+		absPath  = filepath.Join(repoPath, relPath)
+	)
+
+	cfg, err := config.LoadFromYAML(absPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			e.LogPersister.Errorf("deployment config file %s was not found", relPath)
+		}
+	}
```

Comment:
TargetDSP is already loading the configuration for us so you can access that mentions field directly by this

```
ds.GenericDeploymentConfig.Notification.Mentions
```

Reply:
1593b1c